cdk/eks/eks-cdk.sh

```diff
@@ -36,6 +38,22 @@ fi
 
 # Deploy or destroy all stacks in the app
 if [[ "$ACTION" == "deploy" ]]; then
+
+  # update vets service config to use the otlp collector when use-otlp is true
+  MANIFEST_FILE="./lib/manifests/sample-app/vets-service-deployment.yaml"
+  MANIFEST_OTLP_FILE="vets-service-deployment-otlp.yaml"
+  if [[ "$USE_OTLP" == "true" ]]; then
+    if [[ -f "$MANIFEST_OTLP_FILE" ]]; then
+      echo "Replacing $MANIFEST_FILE with $MANIFEST_OTLP_FILE..."
+      cp "$MANIFEST_OTLP_FILE" "$MANIFEST_FILE"
```

Comment:
minor: better to keep the original file `./lib/manifests/sample-app/vets-service-deployment.yaml`. But it's fine as we are going to use the otlp yaml file in the future.